android/quest/build.gradle

```diff
@@ -317,7 +317,7 @@ dependencies {
     androidTestImplementation "androidx.test:core-ktx:$versions.atsl_core"
 
 //    debugImplementation because LeakCanary should only run in debug builds.
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:2.10'
+//    debugImplementation 'com.squareup.leakcanary:leakcanary-android:2.10'
```

Comment:
Why have we disabled this dependency?

Reply:
NVM, I answered myself when I merged this to another branch. 

- There appears to be a duplicate dependency